src/components/ButtonPanel.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import Button from './Button'
+
+function ButtonPanel(){
+  return (
+          
+          <div className="keyPanel">
+          <div className="row">
+            <Button >AC</Button>
+            <Button >+/-</Button>
+            <Button >%</Button>
+            <Button >/</Button>
+          </div>
+          <div className="row">
+            <Button >7</Button>
+            <Button >8</Button>
+            <Button >9</Button>
+            <Button >X</Button>
+          </div>
+          <div className="row">
+            <Button >4</Button>
+            <Button >5</Button>
+            <Button >6</Button>
+            <Button >-</Button>
+          </div>
+          <div className="row">
+            <Button >1</Button>
+            <Button >2</Button>
+            <Button >3</Button>
+            <Button >+</Button>
+          </div>
+          <div className="row">
+            <Button >0</Button>
+            <Button >.</Button>
+            <Button >=</Button>
+          </div>
+          </div>

```

Comment:
From my comment above (regarding the `<Button />` component,

this implies, you should render `<Button />` component here and pass a `name` prop